dot/utils.go

```diff
@@ -187,24 +188,32 @@ func NewTestConfigWithFile(t *testing.T) (*Config, *os.File) {
 
 // ExportConfig exports a dot configuration to a toml configuration file
 func ExportConfig(cfg *Config, fp string) *os.File {
-	var (
-		newFile *os.File
-		err     error
-		raw     []byte
-	)
+	raw, err := toml.Marshal(*cfg)
+	if  err != nil {
+		logger.Error("failed to marshal configuration", "error", err)
+		os.Exit(1)
+	}
+	return WriteConfig(raw, fp)
```

Comment:
I think this should convert the `Config` to `ctoml.Config` otherwise there may be some marshalling/unmarshalling errors

Reply:
Test failures were due to a different issue. This would be `Config` and wouldn't have marshaling failures because non-primitive datatype would be ignored. This code was already there before.